bin/eth-proofs/src/cli.rs

```diff
@@ -52,7 +53,7 @@ impl Args {
             rpc_url: Some(self.http_rpc_url.clone()),
             cache_dir: None,
             custom_beneficiary: None,
-            prove: !self.execute_only,
+            prove_mode: (!self.execute_only).then_some(SP1ProofMode::Groth16),
```

Comment:
```suggestion
            prove_mode: (!self.execute_only).then_some(SP1ProofMode::Compressed),
```

The default here should be compressed proofs, right?

Reply:
Otherwise, proving time will go up.